source/val/validation_state.h

```diff
@@ -290,11 +294,14 @@ class ValidationState_t {
   bool IsStructTypeWithBuiltInMember(uint32_t id) const {
     return (builtin_structs_.find(id) != builtin_structs_.end());
   }
+
  private:
   ValidationState_t(const ValidationState_t&);
 
   const spv_const_context context_;
 
+  const spv_const_validator_options options_;
```

Comment:
Is this always a valid options object?  Must say so one way or the other.
Probably safer to assume it always is.
The constructor should check.